### DISCUSSION TITLE
Is floating mode still unstable ?

### DISCUSSION THREAD

**Discussion by searleser97** [NeogitOrg/neogit#910] [Q&A] [answered]
Does the floating mode now supports popups ?

I tested it and works as expected within the neogit tab, however, I would like to know how to open them as floating windows when calling the popups from other tabs (regular tabs), specifically would like to be able to open the popups as floating windows within the diffview tab

**Comment by searleser97** [comment, marked as answer]
for some unknown reason it now works as expected haha

**Reply by CKolkey** [reply]
Awesome :) I don't use floating windows, but I'm glad to hear they're working
